File: Beecrowd/1192 O jogo Matemático de Paula/main.py
N = int(input())

for i in range(N):
  c = input()
  if c[0] == c[2]:
    cal = int(c[0]) * int(c[2])
  elif c[1].islower():
    cal = int(c[0]) + int(c[2])
  else:
    cal = int(c[2]) - int(c[0])
    
  print(cal)

File: Beecrowd/1192 O jogo Matemático de Paula/main.c
#include <stdio.h>
#include <stdlib.h>
#include <ctype.h>

int main()
{
    int N, i, cal;
    char c[4];

    scanf("%d", &N);

    for (i = 0; i < N; i++)
    {
        scanf(" %s", c);

        if (c[0] == c[2])
            cal = atoi(&c[0]) * atoi(&c[2]);
        else if (islower(c[1]))
            cal = atoi(&c[0]) + atoi(&c[2]);
        else
            cal = atoi(&c[2]) - atoi(&c[0]);

        printf("%d\n", cal);
    }

    return 0;
}